modules/cnv-connecting-resource-bridge-network.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-attaching-vm-multiple-networks.adoc
+
+[id="connecting-resource-bridge-network-{context}"]
+= Connecting a resource to a bridge-based network
+
+As a network administrator, you can configure a NetworkAttachmentDefinition
+of type `cnv-bridge` to provide Layer-2 networking to Pods and virtual machines.
+
+.Prerequisites
+
+* {ProductName} 2.0 or newer
+* A Linux bridge must be configured and attached to the correct
+Network Interface Card on every node.
+* `vlan_filtering` must be enabled on the bridge if you want to use VLANs.
+
+.Procedure
+
+. Create a new file for the NetworkAttachmentDefinition in any local directory.
+The file must have the following contents, modified to match your
+configuration:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: a-bridge-network
+spec:
+  config: '{
+    "cniVersion": "0.3.0",
+    "name": "a-bridge-network", <1>
+    "type": "cnv-bridge", <2>
+    "bridge": "br0", <3>
+    "isGateway": true,
+    "ipam": {
```

Comment:
Please remove the IPAM for L2 networks `"ipam": {}` also please take a look on this issue https://github.com/kubevirt/kubevirt/issues/2163

for Pod that want to have a L3 bridge they need `"ipam": { "type": "dhcp"} but for virtual machines leave the ipam empty so the vm will ask for the dhcp and not the CNI plugin.
if we don't do it the dhcp release and renew will not work.

BTW please don't document the IPAM type host-local this plugins is not for a multi node environment.
the only way for the host-local to be multi node is if the `/mnt/cluster-ipam` folder is mounted and shared by all the nodes but them we can get a rise condition

Reply:
Thanks for the explanation! I have very minimal networking knowledge so I appreciate it. :) I'm changing it to just `"ipam": {}` for this YAML.